Editor/src/Editor.cpp

```diff
@@ -105,7 +98,8 @@ class ExampleLayer : public Light::Layer
 		ImGui::PopStyleVar();
 
 		ImGui::Begin("Scene Settings");
-		ImGui::DragFloat3("Light Position", &(m_lightPos.x), 0.01f);
+		glm::vec3 v=m_scene.getlightPos();
+		ImGui::DragFloat3("Light Position", &(v.x), 0.01f);
```

Comment:
Instead of directly modifying the returned value of `getLightPos` (which is unlikely to do anything since it acts on a temporary return value), get the value into `v` and then set `v` using `setLightPos()`